src/components/MetadataContainer/MetadataContainer.tsx

```diff
@@ -88,7 +88,7 @@ export default function MetadataContainer({
         {!!level?.data && (
```

Comment:
Same in this file: landingPageCopy doesn't need to be optional anymore, right?

Reply:
There's lots of unused imports in this file as well, please remove them

Reply:
I will remove them